src/mltk/predictor/gam/GA2MLearner.java

```diff
@@ -68,6 +69,9 @@
 
 		@Argument(name = "-l", description = "learning rate (default: 0.01)")
 		double learningRate = 0.01;
+		
+		@Argument(name = "-p", description = "maximum number of pairs")
```

Comment:
I would not add this parameter since the input pairs is required and specified by '-I'. If called from other program, the caller should have a sense how many pairs to use and provide the list of pairs.